### PR TITLE
don't miss out on JS events if GA is blocked

### PR DESCRIPTION
**Pull request by sravfeyn**
http://manage.dimagi.com/default.asp?170058#956037 
@esoergel 
cc @nickpell @NoahCarnahan (pinging you both, as this affects almost all of GA stuff)

### CODE REVIEW

**Comment by esoergel** [comment]
:star2:

**Comment by benrudolph** [comment]
nice. also, just flagging, we miss all our firefox events because firefox doesn't use `window.event` at some point we should refactor so that we use the event passed in from function which is supported in all modern browsers

**Comment by benrudolph** [comment]
take that back. we still log them, but the functionality is slightly changed since we don't prevent default. sorry about that false alarm!

File: corehq/apps/style/templates/style/includes/analytics_google.html [hunk 1]
```diff
@@ -91,17 +91,26 @@
...
         }
-        // check the last from the right
-        // rather than the 3rd or 4th from the left, because it's variable
-        // https://developers.google.com/analytics/devguides/collection/analyticsjs/events#overview
-        var lastArg = arguments[arguments.length - 1];
-        if (lastArg && lastArg.hitCallback && window.event) {
-            window.event.preventDefault();
+        else{
+            // If GA is blocked, run the hitCallback
+            var callBack = lastArg.hitCallback;
```
Comment (esoergel) [inline corehq/apps/style/templates/style/includes/analytics_google.html]: `lastArg` is undefined at this point.

Comment (sravfeyn) [inline corehq/apps/style/templates/style/includes/analytics_google.html]: ugh! It worked and I didn't pay attention, will fix that (wondering how it worked)

**Pull request merged**